src/bitter/lsb0_reader.hpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Steinwurf ApS 2016.
+// All Rights Reserved
+//
+// Distributed under the "BSD License". See the accompanying LICENSE.rst file.
+#pragma once
+
+#include "reader.hpp"
+
+namespace bitter
```

Comment:
Why have this typedef?

Reply:
It is just a convenience alias, in many cases you would simply use one or the other. So no need for the template argument. In cases where you need a reader that can work in both modes you can use the reader class directly.